packages/docs/docs/communications/organizing-communications/organizing-communications.md

```diff
@@ -0,0 +1,167 @@
+
+The `Communication.topic` element is a description of the content of the communication. It’s easy to think of this as the subject line of an email. Keeping communications well organized by topic can assist with billing and driving workflows. For example, if a patient consults with a physician via phone, this communication can be given a topic of, for example `phone-consult`. Being able to see this topic allows for easy billing to the patient. Additionally, a communication with a `prescription-refill-request` topic is an easy indicator to review this request and fulfill it if necessary.
+
+Organizing communications via threading is an effective way to keep related messages grouped together. In healthcare apps, threads can include SMS chains, in-app chats, email exchanges, and more. These threads can constitute an [encounter](/docs/communications/async-encounters/async-encounters), which can be represented on `Communication.encounter`. The `Communication.encounter` element references a specific encounter that is closely related to the communication, or during which the communication took place. In addition, you can use the `Communication.inResponseTo` element. This element is a an array of references to `Communication` elements that preceded the current one and can be useful for tying all the messages in a threads together. Threading communications in this way allows for users to easily view all messages relevant to a single encounter.
+
+## Codesystems
+
+It is recommended to use LOINC or SNOMED codes to classify `Communication.category` and `Communication.topic`, but it is also possible to use custom coding internal to your practice.
+
+Below is an example of organizing a communication using all three of LOINC, SNOMED, and a custom code.
+
+```
+{
+  resourceType: "Communication",
+	//...
+	category: [
+	  // A category coded using LOINC
+		{
+			text: "Alert",
+			coding: [
+				{
+					code: "74018-3"
+					system: "https://fhir.loinc.org",
+				}
+			]
+		},
+	],
+	//...
+	topic: {
+	  // A topic coded using SNOMED
+		text: "High blood pressure",
+		coding: [
+			{
+				code: "38341003",
+				system: "http://snomed.info/sct",
+				display: "Hypertensive Disorder"
+			}
+		]
+	}
+}
+
+{
+	resourceType: "Communication",
+	//...
+	// An example of how an internal custom coding could be used for both category and topic
+	category: [
+		{
+			text: "Appointment Reminder",
+			coding: [
+				{
+					code: "apt-rem",
```

Comment:
perhaps make this a full word like `appointment-reminder` for clarity.  Same with phicial example below.